Tasks/Loktionova/Task2/task2.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+
+mode = input ('Do you want to know the current time? Print Yes if yes?: ')
+if mode == 'Yes':
+    time = datetime.datetime.now().strftime('%H:%M').split(':')
+else:
+    time = input ('Enter your time in format HH:MM: ').split(':')
+hour = int(time[0])
+min = int(time[1])
```

Comment:
where at least some validation? you should at least check for the string length and then for the ':' symbol to be included in the input, -1 point for the unwise choice